gitsome/config.py

```diff
@@ -643,6 +652,11 @@ def save_config(self):
             parser.set(self.CONFIG_SECTION,
                        self.CONFIG_VERIFY_SSL,
                        self.verify_ssl)
+            # Update existing config files without enable_avatar correctly
```

Comment:
Let's remove this comment, the code is self-documenting.

Reply:
sure, see 34defb9603cd887ccbac2ce1166ef62bfba04ec5